contrib/vagrant/scripts/03-install-kubernetes-worker.sh

```diff
@@ -335,7 +335,7 @@ ExecStart=/usr/bin/kubelet \\
   --kubeconfig=/var/lib/kubelet/kubelet.kubeconfig \\
   --fail-swap-on=false \\
   --make-iptables-util-chains=false \\
-  --node-ip=${node_ip} \\
+  --node-ip=$([[ "$node_ip" == "$node_ipv6" ]] && echo $node_ip || echo "$node_ip,$node_ipv6") \\
```

Comment:
I'm not sure I understand the condition here. Won't `$node_ipv6` be empty in case IPv6 is disabled?

Reply:
Right, we're dealing with 3 situations here:
1. `$node_ipv6` is empty, and I just pushed a patch for it, please see the latest update
2. `$node_ip` == `$node_ipv6`, this is [possible](https://github.com/cilium/cilium/blob/master/Vagrantfile#L219-L221) when `IPV6_EXT=1`
3. `$node_ip` != `$node_ipv6`: we concat two vars only when this happens

Reply:
OK. Thanks for the explanation! I think it would be worth adding a small comment for the `node_ip_addresses` function to explain the 2nd case.